pkgs/tools/misc/thumbs/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "thumbs";
+  version = "0.7.1";
+
+  src = fetchFromGitHub {
+    owner = "fcsonline";
+    repo = "tmux-thumbs";
+    rev = version;
+    sha256 = "sha256-PH1nscmVhxJFupS7dlbOb+qEwG/Pa/2P6XFIbR/cfaQ=";
+  };
+
+  cargoSha256 = "sha256-6htKiXMMyYRFefJzvDnmdx3CJ3XL8zONhGlV2wcbr9g=";
+
+  patches = [ ./fix.patch ];
```

```suggestion
  patches = [ ./fix.patch ];

```
You are sure this does not need to be cargoPatches?

---

Both options work in that case, although it `cargoPatches` seems to be more suitable in that place.